### PR TITLE
Modified nj_eitc_eligibility to use AGI

### PR DESCRIPTION
**Pull request by jachan**
## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/200

## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
Small change to update income check of EITC. It previously used `total_federal_income` from the Direct File import, swapped it to use `federal_agi`

## How to test?
Test Case 1
- Select `childless eitc` persona
- Answer NO to the "In 2024, could you be someone else's qualifying child for the Earned Income Tax Credit?" question
- Validate that Line 58 of the Tax returns has a 240$ Credit for EITC

Test Case 2
- Select `childess eitc` persona
- Click on the " let me edit the response XML" button
- Set AGI to 18591 (above the EITC income limit)
- Validate that Line 58 of the Tax returns does not have a 240$ Credit for EITC

### CODE REVIEW

File: app/lib/efile/nj/nj_flat_eitc_eligibility.rb [hunk 1]
```diff
@@ -57,11 +57,11 @@ def all_outside_fed_eitc_age_range?(*ages)
         end
 
         def is_under_income_total_limit?(intake)
-          if intake.filing_status_mfj?
-            intake.direct_file_data.fed_income_total < 25_511
-          else
-            intake.direct_file_data.fed_income_total < 18_591
-          end
+          intake.direct_file_data.fed_agi < if intake.filing_status_mfj?
```
Comment (mmazanec22) [inline app/lib/efile/nj/nj_flat_eitc_eligibility.rb:60]: [dust] Assigning the if clause to a `threshold` variable here would make this more readable

**Review by mmazanec22 — APPROVED** [review]

File: app/lib/efile/nj/nj_flat_eitc_eligibility.rb (hunk 1 above)
Comment (mpidcock) [reply app/lib/efile/nj/nj_flat_eitc_eligibility.rb:60]: agree!

**Review by mpidcock — APPROVED** [review]

File: app/lib/efile/nj/nj_flat_eitc_eligibility.rb (hunk 1 above)
Comment (jachan) [reply app/lib/efile/nj/nj_flat_eitc_eligibility.rb]: Done, appreciate the suggestion!

**Review by mluedke2 — COMMENTED** [review]
👍

**Pull request merged**